src/nodejs/monitor/cli/new_transaction_alert_loop.js

```diff
@@ -102,35 +108,41 @@ if (defly) print('sleeping: ' + program.wait + ' s')
 
 let last_run_result = ''
 
+get_last_transaction(argz)
+
 const intervalObj = setInterval(() => {
   if (i > program.loop && program.loop !== 0) {
     clearInterval(intervalObj)
     print('clearing timer')
   } else {
     print('loop #: ' + i + ' of ' + (program.loop ? program.loop : 'infinity'))
-
-    get_last_transactions_by_address.run(argz).then((r) => {
-      let message = {
-        to: to,
-        subject: program.Subject + ' for ' + address,
-        body: program.body
-      }
-
-      let rstr = message.body = dbg.lookDeep(r)
-
-      if (defly) dbg.logDeep('body: ', message.body)
-
-      if (last_run_result && last_run_result !== rstr) {
-        print('New Transaction')
-        email.send(message).then((id) => {
-          print('message away: ' + id)
-        })
-      }
-      last_run_result = rstr
-      if (defly) print('last result: ' + last_run_result)
-    })
+    get_last_transaction(argz)
   }
 
   print('sleeping: ' + program.wait + ' s')
   i++
 }, program.wait * 1000)
+
+
+function get_last_transaction(argz) {
```

Comment:
It might be best to rename `argz` to something else as it can be confused as it's already declared in an upper scope

Reply:
Thank you. Good point, will address in next PR. I need to clean up the sub line too anyway.